.gitignore

```diff
@@ -7,3 +7,6 @@ __pycache__/
 *.egg-info/
 /cafebabel/static/uploads/
 /data/
+.idea/
+.DS_Store
+venv/
```

Comment:
Do you still have issues with your gitignore? These should not be here but in you ~/gitignore as they are specific to your personal environment.

Reply:
oui ça ne marche pas quand je mets ces fichiers dans .gitignore à la racine de ma machine, mais ça fonctionne quand je les mets dans le .gitignore du projet. Donc c'est ce que j'ai fait

Reply:
Peut-être séparer ça dans une section qui explique pourquoi on a ça ici, pour qu'on comprenne bien que c'est spécifique à toi. Ca parait évident comme ça, mais dans un an ça le sera moins…

```

# Johan's speific configuration (workaround as it does not work when stored in ~/gitignore)
/.idea/
.DS_Store
/venv/
```

Reply:
ok